### PR TITLE
Update UI templates for S3 and Snapshot datasets

### PR DESCRIPTION
**Pull request by sreevatsanraman**
* Update UI templates for S3Avro, S3Parquet, SnapshotAvro, SnapshotParquet
* Make pathFormat optional and fix description
screen shot 2015-09-20 at 10 58 16 am
screen shot 2015-09-20 at 10 58 33 am

### CODE REVIEW

File: cdap-ui/templates/cdap-etl-batch/S3Avro.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "display" : "S3 Sink - Avro",
+      "position" : [ "accessID", "accessKey", "basePath", "pathFormat"],
+      "fields" : {
+        "accessID" : {
+          "widget": "textbox",
+          "label": "Access ID"
+        },
+
+        "accessKey" : {
+          "widget": "textbox",
```
Comment (bdmogal) [inline cdap-ui/templates/cdap-etl-batch/S3Avro.json]: should this be a password field?

File: cdap-ui/templates/cdap-etl-batch/S3Parquet.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "display" : "S3 Sink - Parquet",
+      "position" : [ "accessID", "accessKey", "basePath", "pathFormat"],
+      "fields" : {
+        "accessID" : {
+          "widget": "textbox",
+          "label": "Access ID"
+        },
+
+        "accessKey" : {
+          "widget": "textbox",
```
Comment (bdmogal) [inline cdap-ui/templates/cdap-etl-batch/S3Parquet.json]: ditto

**Comment by bdmogal** [comment]
couple quick questions. otherwise lgtm

File: cdap-ui/templates/cdap-etl-batch/S3Avro.json (hunk 1 above)
Comment (sreevatsanraman) [inline cdap-ui/templates/cdap-etl-batch/S3Avro.json]: No people should be able to see the accessKey. It is generated by S3 and not something user selects.

File: cdap-ui/templates/cdap-etl-batch/S3Parquet.json (hunk 2 above)
Comment (sreevatsanraman) [inline cdap-ui/templates/cdap-etl-batch/S3Parquet.json]: same comment as above :)

**Pull request merged**